cadence/templates/server-configmap.yaml

```diff
@@ -108,6 +118,11 @@ data:
           prometheus:
             timerType: {{ default .Values.server.metrics.prometheus.timerType .Values.server.matching.metrics.prometheus.timerType }}
             listenAddress: "0.0.0.0:9090"
+          {{- if or .Values.server.metrics.statsd .Values.server.matching.metrics.statsd }}
+            statsd:
```

Comment:
@aoby @sagikazarmark if I'm understanding the cadence examples correctly I think this needs to be unindented so `statsd` is under `metrics` and not `prometheus`.

That yaml this generates looks like this:
```
      prometheus:
        timerType: histogram
        listenAddress: "0.0.0.0:9090"
        statsd:
          hostPort: "my-statsd-host:8125"
          prefix: "cadence.worker"
```

Reply:
Oh wow good catch! I inspected the generated yaml and didn't notice that, and I haven't yet deployed this new chart.

Yes, that's correct - the `statsd` should be under `metrics`. I'll send an update PR today

Reply:
No problem, I created a PR here: https://github.com/banzaicloud/banzai-charts/pull/1101

Reply:
Awesome, thanks @bjacobso